### ISSUE TITLE
Add icon for charging station

### ISSUE THREAD

**Issue by PinguDEV-original** [Qwant/erdapfel#1469]
Please add an icon for charging stations. As you can see [here](https://www.qwant.com/maps/place/osm:node:10294793142@Sankt_Josef_Hospital), there is no real icon (like there is an icon, but not something like [this](https://pixabay.com/vectors/gas-station-symbol-e-car-3886561/)).

**Comment by sdrll** [comment]
I agree, this would be useful. But, want to fix opening hours and refresh old data first
